new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/Operator.java

```diff
@@ -3,69 +3,10 @@
 
 package oracle.weblogic.kubernetes.actions.impl;
 
-import io.kubernetes.client.openapi.ApiException;
 import oracle.weblogic.kubernetes.actions.impl.primitive.Helm;
-import oracle.weblogic.kubernetes.extensions.LoggedTest;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-public class Operator implements LoggedTest {
-
-  /**
-   * The URL of the Operator's Helm Repository.
-   */
-  //Question - is this URL correct?
-  private static String OPERATOR_HELM_REPO_URL = "https://oracle.github.io/weblogic-kubernetes-operator/charts";
-
-  /**
-   * The name of the Operator's Helm Chart (in the repository).
-   */
-  private static String OPERATOR_CHART_NAME = "weblogic-operator/weblogic-operator";
-
-  /**
-   * Install WebLogic Kubernetes Operator.
-   *
-   * @param params operator parameters for helm values
-   * @return true if the operator is successfully installed, false otherwise.
-   * @throws ApiException - if Kubernetes client API call fails
-   */
-  public static boolean install(OperatorParams params) throws ApiException {
-
-    String namespace = params.getNamespace();
-    String serviceAccount = params.getServiceAccount();
-
-    // assertions for required parameters
-    assertThat(namespace)
-        .as("make sure namespace is not empty or null")
-        .isNotNull()
-        .isNotEmpty();
-
-    assertThat(params.getReleaseName())
-        .as("make sure releaseName is not empty or null")
-        .isNotNull()
-        .isNotEmpty();
-
-    boolean success = false;
-    if (new Helm().chartName(OPERATOR_CHART_NAME).repoUrl(OPERATOR_HELM_REPO_URL).addRepo()) {
-      logger.info(String.format("Installing Operator in namespace %s", namespace));
-      success = new Helm().chartName(OPERATOR_CHART_NAME)
-                        .releaseName(params.getReleaseName())
-                        .namespace(namespace)
-                        .values(params.getValues())
-                        .install();
-    }
-    return success;
-  }
-
-  public static boolean upgrade(OperatorParams params) {
-    return true;
-  }
 
+public class Operator extends Helm {
```

Comment:
i need to take a much closer look at this, my gut reaction is that extension here is not the best solution

Reply:
seems like an operator is much more than just a set of helm params
maybe we could `implement HelmRelease` 
or have a variable that is a `HelmParams` 

i think an action impl should *never* extend a primitive
they should *use* primitives, not "be" them... imo

Reply:
Agree.  HelmParams or the other choice seems more appropriate.

Reply:
Agree. Removed inheritance and modified impl classes to call Helm instead.